packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js

```diff
@@ -37,6 +32,12 @@ export default class TrafficController {
       this.services = data;
       return data;
     });
+    if (this.getSubnet()) {
+      this.subnet = this.getSubnet();
+      this.selectedIp = this.subnet;
+      this.model = this.selectedIp;
```

Comment:
Why you need 3 variables with same value ?

Reply:
One for the select list display into the html, one for the call API and the last for the model else it doesn't work.